src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] currentTable;
+    private int currentCapacity;
+    private int threshHold;
```

Comment:
I think currentCapacity variable is redundant, you can use DEFAULT_CAPACITY to create your table and then use table.length instead of currentCapacity